iphone/Maps/Core/TextToSpeech/MWMTextToSpeech.mm

```diff
@@ -286,16 +288,12 @@ + (void)removeObserver:(id<MWMTextToSpeechObserver>)observer {
 
 namespace tts
 {
-std::string translatedTwine(std::string const & twine)
+std::string translateLocale(std::string const & localeString)
 {
-  auto const & list = routing::turns::sound::kLanguageList;
-  auto const it =
-      find_if(list.begin(), list.end(),
-              [&twine](std::pair<std::string, std::string> const & pair) { return pair.first == twine; });
-
-  if (it != list.end())
-    return it->second;
-  else
-    return "";
+  NSString *nsLocaleString = [NSString stringWithUTF8String: localeString.c_str()];
+  NSLocale *locale = [[NSLocale alloc] initWithLocaleIdentifier: nsLocaleString];
+  NSString *localizedName = [locale localizedStringForLocaleIdentifier:nsLocaleString];
```

Comment:
I'd love to write a test but is there a way to call this method from Swift oder do you suggest a test in Objective-C?

Reply:
Swift will not be working with the cpp (until we upgrade to the latest xcode and up the target to ios 15 because there will no way back to the xcode 14 - https://www.swift.org/documentation/cxx-interop/#enabling-c-interoperability).
So the only way to write tests for this method is objc.

Reply:
Ok I wrote some tests in Objective-C. It was easier than expected...

I also moved the test classes to the same group as their respective classes in the project. The Tests group basically mimics the project folder structure:

<img width="350" alt="image" src="https://github.com/organicmaps/organicmaps/assets/6050639/86dc986e-f7b8-4fdf-af6b-f39001a66254">

I do this in my large projects to organize my test files but I don't know if you like it?